patches/react-native+0.48.0.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/node_modules/react-native/Libraries/Alert/AlertIOS.js b/node_modules/react-native/Libraries/Alert/AlertIOS.js
```

Comment:
beware of a rebase here, it'll need bumping to 0.48.4

Reply:
Yup, indeed, gonna do so now. We can also move those `sed` scripts to these patches btw.

Reply:
so good